assets/index.html

```diff
@@ -8,14 +8,21 @@
   <link rel="shortcut icon" type="image/x-icon" href="../assets/favicon.png" />
   <link rel="icon" type="image/x-icon" href="../assets/favicon.png" />
   <body>
-    <div class="leaderboard">
-      <img
-        class="leaderboard__img"
-        src="../assets/LF_Opensource.svg"
-        alt="Leapfrog Open Source Logo"
-      />
-      <h1>Frogtoberfest Leaderboard 2023</h1>
-      <div class="leaderboard__table">{table}</div>
+    <div>
+      <nav>
+        <img src="../assets/LF_Opensource.svg" alt="Logo">
+        <div class="nav-links">
+          <a href="https://frogtober.lftechnology.com/#participation">Participation</a>
```

Comment:
Can we use the frogtoberfest URL please? The other one frogtober.lftechnology.com is likely an alias.

```suggestion
          <a href="https://frogtoberbest.lftechnology.com/#participation">Participation</a>
```